Minesweeper.py

```diff
@@ -44,132 +35,133 @@ def __init__(self):
+    """
     def __init__(self):
         self.image = pygame.transform.scale(pygame.image.load('reset_button.png'), (Tiles.TILE_SIZE, Tiles.TILE_SIZE))
         self.rect = self.image.get_rect(center=(win.get_width() // 2, win.get_height() - 30))
 
     def draw(self):
+        """displays the restart button image
+        """
         win.blit(self.image, self.rect)
 
     def handle_click(self, pos):
+        """ calls the restart function when the image is clicked
+        """
         if self.rect.collidepoint(pos):
             game.restart()
 
-
-# Define the Game class to manage the game loop and game objects
 class Game(object):
+    """ manages the main game features such as the restart function and the main game loop"""
     def __init__(self):
-        # DHB: initialising a library should not be done in an object initialiser like this. Move it to file scope, or move everything on file scope into this class (which would be a bad choice)
-        pygame.init()
-        # DHB: you're not really use the game_objects system... use it, or remove it  
         self.game_objects = []
-        # DHB: it is important to comment all member variables, explaining what they are used for and what they can contain, just like all functions and classes.
         self.tiles = None
         self.game_over = False
         self.game_won = False
-        self.game_won_message_printed = False
-        # DHB: for a state variable like this, it's especially important to ensure that the valid states are well defined. You could for example use a game state
-        # DHB: enumeration class, similar to the color class, to define the valid status
         self.state = "tutorial"
         self.clock = pygame.time.Clock()
         self.time_elapsed = 0
         self.start_time = 0
+        pygame.mouse.set_visible(True)
+        pygame.mouse.set_cursor(*pygame.cursors.tri_left)
 
         self.tutorial_screen = TutorialScreen()
         self.game_won_screen = GameWonScreen()
         self.reset_button = ResetButton()
 
     def add_game_objects(self, *list_of_game_objects, **dict_of_game_objects):
+        """ adds amount of game objects
+        """
         for game_object in list_of_game_objects:
             self.add_game_object(game_object)
         for game_object_id, game_object in dict_of_game_objects.items():
             self.add_game_object(game_object)
 
     def add_game_object(self, game_object):
+        """adds individual game object
+        """
         self.game_objects.append(game_object)
         if isinstance(game_object, Tiles):
             self.tiles = game_object
 
-    def get_game_objects_with_attribute(self, attribute_name):
-        return [game_object for game_object in self.game_objects if hasattr(game_object, attribute_name)]
-
-    def draw(self): # calls all of the draw functions
+    def draw(self):
+        """calls the draw functions required under certain circumstances
+        """
```

Comment:
doing the background fill should likely be the first thing done in this function